app/controllers/audits/allocations_controller.rb

```diff
@@ -74,7 +74,11 @@ def redirect_params
     end
 
     def redirect_url
-      audits_allocations_url(redirect_params)
+      if params[:redirect_to_content_item] && content_ids.count == 1
```

Comment:
What is the reason behind ```content_ids.count==1```?

Reply:
In theory to avoid us ever trying to pass `redirect_to_conent_item` `true` but also passing either no content Ids (`500` error) or multiple Ids, because that wouldn't make any sense. No strong feelings on this logic. If you prefer I can instead check `content_ids.positive?`?

Reply:
I suppose it would be most restful to return a `400` if someone sends `redirect...` `true` and a non-unity number of Ids?

Reply:
As discussed, we'll leave this as-is for now.